python/lsst/ts/tunablelaser/component.py

```diff
@@ -138,12 +157,30 @@ async def set_burst_count(self, count):
             The amount to pulse the laser.
         """
         await self.m_cpu800.set_burst_count(count)
+        await self.csc.evt_burstCountSet.set_write(count=count)
 
     async def start_propagating(self):
         """Start propagating the beam of the laser."""
         await self.m_cpu800.start_propagating()
+        await asyncio.sleep(10)  # laser warmup delay
```

Comment:
I strongly suggest making this delay a named constant (or a CSC configuration parameter if we are likely to need to tune it).